streaming-binlog/src/main/java/org/openmrs/analytics/DebeziumListener.java

```diff
@@ -16,18 +16,20 @@
 
 import java.io.IOException;
 
-import ca.uhn.fhir.context.FhirContext;
-import com.beust.jcommander.JCommander;
-import com.beust.jcommander.Parameter;
-import com.beust.jcommander.Parameters;
-import com.google.common.annotations.VisibleForTesting;
 import org.apache.camel.CamelContext;
 import org.apache.camel.LoggingLevel;
 import org.apache.camel.Service;
 import org.apache.camel.builder.RouteBuilder;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.beust.jcommander.JCommander;
```

Comment:
Why the `import` order is changed here? Did you run `mvn clean package` before sending the PR?